### PR TITLE
Document standalone ops for 1.12.0

### PR DESCRIPTION
**Pull request by RandySheriffH**
Document the feature of "standalone ops".
By which custom ops could call directly into a native op to utilize its functions with customization.

### CODE REVIEW

File: docs/reference/operators/add-custom-op.md [hunk 1]
```diff
@@ -36,6 +36,20 @@ A new op can be registered with ONNX Runtime using the Custom Operator API in [o
...
+
+Steps are:
+
+1. Create a native operator instance by providing type constraints and attributes.
+2. Call the operator by feeding inputs and outputs.
+3. Recycle the operator.
+
+Please see examples [here](https://github.com/microsoft/onnxruntime/blob/ced7c2deac958391414d2bbf951f86e2fc904b05/onnxruntime/test/shared_lib/custom_op_utils.cc#L210).
+
+Note that the native op will be created on the same executioin provider as the custom op.
```
Comment (faxu) [inline docs/reference/operators/add-custom-op.md:50]: typo on "execution"

File: docs/reference/operators/add-custom-op.md [hunk 2]
```diff
@@ -36,6 +36,20 @@ A new op can be registered with ONNX Runtime using the Custom Operator API in [o
...
+With the feature, custom ops are empowered by native operators not only for functionalities but also for customizations like state management, or pre-processing of inputs.
```
Comment (faxu) [inline docs/reference/operators/add-custom-op.md:51]: Can you clarify this line? Also, it'll be useful to highlight the benefits of this feature (including this) at the top of the section.

**Review by faxu — CHANGES_REQUESTED** [review]

File: docs/reference/operators/add-custom-op.md (hunk 1 above)
Comment (RandySheriffH) [reply docs/reference/operators/add-custom-op.md]: Resolved.

File: docs/reference/operators/add-custom-op.md (hunk 2 above)
Comment (RandySheriffH) [reply docs/reference/operators/add-custom-op.md]: Updated.

**Review by faxu — APPROVED** [review]

**Pull request merged**